build/AzurePipelineTemplates/CsWinRT-FunctionalTest-Steps.yml

```diff
@@ -62,20 +62,20 @@ steps:
         TargetFolder: $(StagingFolder)\FunctionalTests\${{ functionalTest }}\
 
     - task: MSBuild@1
-      displayName: Publish ${{ functionalTest }} for AOT (net8.0)
+      displayName: Publish ${{ functionalTest }} for AOT (net9.0)

```

Comment:
Should we also keep the .NET 8 tests, since that's still the supported LTS release?

Reply:
I was thinking about that, but for AOT, I thought .NET 9 tests are basically similar if not stricter and we had some .NET 9 AOT fixes.  We still have .NET 8 trimming tests.